src/dropdown.ts

```diff
@@ -2556,16 +2591,6 @@ export default class Dropdown extends LitElement implements FormControl {
           this.#inputElementRef.value.clientWidth;
       }
     }
-
-    // Dropdown's internal label now needs to be updated to reflect the now enabled
-    // or disabled selected option(s). `this.internalLabel` uses the `this.lastSelectedOption`
-    // getter, whose return value is derived from the state of another component: Dropdown
-    // Option. That component's state is reactive. But a change to it won't trigger a
-    // rerender of this component. Same deal for multiselect Dropdown's tags.
-    //
-    // If multiselect, the `areSomeOptionsSelected` and `areAllOptionsSelected` getters need
-    // to rerun so Select All is updated. Dropdown's tags also need ot be updated.
-    this.requestUpdate();
```

Comment:
No longer needed now that `this.selectedAndEnabledOptions`, which triggers a rerender, is set.